packages/web-components/src/keplr-connection/errors.js

```diff
@@ -0,0 +1,7 @@
+// @ts-check
+
+export const Errors = {
+  enableKeplr: '0',
+  networkConfig: '1',
+  noSmartWallet: '2',
```

Comment:
Suggest more searchable constants:
```suggestion
  enableKeplr: 'KEPLR_CONNECTION_ERROR_ENABLE_KEPLR',
  networkConfig: 'KEPLR_CONNECTION_ERROR_NETWORK_CONFIG',
  noSmartWallet: 'KEPLR_CONNECTION_ERROR_NO_SMART_WALLET',
```

Reply:
Done